R-package/configure

```diff
@@ -1,6 +1,6 @@
 #! /bin/sh
 # Guess values for system-dependent variables and create Makefiles.
-# Generated by GNU Autoconf 2.69 for lightgbm 2.3.2.
```

Comment:
by the way,  this  file  is automatically generated and should  never be  updated manually:  https://github.com/microsoft/LightGBM/blob/master/R-package/README.md#preparing-a-cran-package-and-installing-it